### DISCUSSION TITLE
Feedback on `deftest` + `testing`

### DISCUSSION THREAD

**Discussion by frenchy64** [frenchy64/fully-satisfies#4] [General]
https://frenchy64.github.io/fully-satisfies/latest/io.github.frenchy64.fully-satisfies.uncaught-testing-contexts.html

**Comment by eggsyntax** [comment]
I think it'd be a terrific addition! The one possible improvement that springs to mind is that when reporting multiple contexts as in your example
```
; please, just tell me what went wrong please
```
it'd be better to have a newline between contexts, so that the above would become
```
; please, just tell me what went wrong 
; please
```
or even, in the best of all possible worlds,
```
; please, just tell me what went wrong 
; ; please
```
to indicate the nested context.

Regardless, thanks for proposing it, I'd love to see it become part of `clojure.test` 🙏

**Reply by frenchy64** [reply]
Thanks @eggsyntax. In terms of the formatting, it seemed like the most natural (mergable) choice to just use the same method of formatting as clojure.test normally does via [`t/testing-contexts-str`](https://github.com/frenchy64/fully-satisfies/blob/b852d5b70567edd68f692559eef0ad69dc0529eb/src/io/github/frenchy64/fully_satisfies/uncaught_testing_contexts.clj#L75-L76).
